students/vkis/lesson03/mailroom.py

```diff
@@ -0,0 +1,129 @@
+# Lession 03 - Mailroom v1
+# =================================================================
+
+# Existing Data (format: 15 char names, 15 char numbers per line)
+rawtext =  """Tony Stark             $453.02
+              Captain America      $1,250.00
+              Daisy Johnson            $1.99
+              Melinda May            $500.01
+              Phil Coulson         $9,999.99
+              Daisy Johnson            $1.99
+              Daisy Johnson           $10.99
+              Melinda May             $55.01
+              Tony Stark             $453.02
+              Captain America      $3,250.00"""
```

Comment:
I sort of can't believe you did this with just a raw text string.  Very cool!

Reply:
:)